packages/zoe/src/contracts/mintAndSellNFT.js

```diff
@@ -85,14 +85,15 @@ const start = zcf => {
           .then(sellItemsCreatorSeat => {
             return harden({
               sellItemsCreatorSeat,
-              sellItemsCreatorFacet: creatorFacet,
+              sellItemsCreatorFacet: /** @type {SellItemsCreatorFacet} */ (creatorFacet),
```

Comment:
I'm not fond of having to mix type goo into our code. can this be moved out of the method code?

Reply:
It is in #1488